server/src/utils/fight/fightMethods.ts

```diff
@@ -655,17 +661,11 @@ const registerHit = (
   updateStats(stats, fighter.id, 'hits', 1, fighter.master);
 };
 
-const dropShield = (fightData: DetailedFight, fighter: DetailedFighter) => {
+const dropShield = (fighter: DetailedFighter) => {
   // Remove brute's shield
   fighter.shield = false;
   fighter.skills = fighter.skills.filter((sk) => sk.name !== SkillName.shield);
   fighter.block -= SkillModifiers[SkillName.shield][FightStat.BLOCK]?.percent ?? 0;
-
-  // Add dropShield step
-  fightData.steps.push({
-    a: StepType.DropShield,
-    b: fighter.index,
-  });
```

Comment:
It would be better to keep the step addition in the dropShield method to avoid code repetition every time we drop the shield.
For the code handling throwing of the shield at the opponent, you can simply not use the dropShield method and edit the fighter props accordingly, or add a parameter to dropShield to let it know to add a step or not